acoustid/config.py

```
@@ -155,18 +155,18 @@ def __eq__(self, other):
             and self.pool_timeout == other.pool_timeout
         )
 
-    def create_url(self):
-        # type: () -> URL
-        kwargs = {}  # type: Dict[str, Any]
-        kwargs["username"] = self.user
-        kwargs["database"] = self.name
+    def create_url(self) -> URL:
+        kwargs: Dict[str, Any] = {
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)